.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: Auth0 Community
-    url: https://community.auth0.com/c/sdks/5
+    url: https://community.auth0.com
     about: Discuss this SDK in the Auth0 Community forums
-  - name: Library Documentation
-    url: https://github.com/auth0/auth0-java-mvc-common/blob/master/README.md
-    about: Read the library documentation
```

Comment:
With the introduction of the EXAMPLES.md file, the README no longer contains all (or even a significant part) of the library docs.